### ISSUE TITLE
ME System Item Duplication

### ISSUE THREAD

**Issue by SuperSaiyanSolar** [AppliedEnergistics/Applied-Energistics-2#2800]
#### Description
Items are being duplicated from the ME system whenever items are shift-clicked in certain circumstances. I'm curious as to whether or not this is a bug or a mod incompatibility, so any help would be appreciated.
#### Environment
-Fully operational ME System with a crafting terminal.
-Using an EnderIO Stirling Generator and an Energy Acceptor for power.
-Items are shift-clicked into the ME system through the crafting terminal or items are stored from said crafting terminal (clicking the "store items" button, specifically).
-Upon shift-clicking, the item can be duplicated indefinitely by continuing to shift-click.
-Bug does not occur when normally placing items in or shift-clicking to remove items
#### Additional Info
-Issue occurs on both a server and in singleplayer
-The list of mods is very extensive, but can be provided if this issue cannot be replicated using solely EnderIO and AE2
-Happens with any item, modded or Vanilla
#### Screenshots and Gifs
Shift-Clicking an item into system: http://i.imgur.com/Hl9zndV.gifv
ME System Setup: http://i.imgur.com/ZJ7efiI.png
When clicking to store items from crafting grid: http://i.imgur.com/cNuNW8P.gifv

- Minecraft Version: 1.10.2
- AE2 Version: RV4-Alpha-11
- Forge Version: Forge 1.10.2-12.18.3.2221
-EnderIO Version: 1.10.2-3.1.161

**Comment by VT-14** [comment]
I'm unable to reproduce this issue with mostly the same mod list (only adding JEI and updating EnderIO to 3.1.169).

Likely another mod or a server plugin causing problems.  Please provide a full mod list.  If you are on a server with special plug-ins, like Sponge, then please note that AE2 does not support them (They fairly often cause these kinds of problems, and supporting one could/would cause demands to support several other variations; which the devs don't have time for).

**Comment by SuperSaiyanSolar** [comment]
We found out the issue and I deeply apologize. It's a mod that we've had for some time that has lately become very broken with most other mods, but we neglected removing the mod for some time because it wasn't causing any game-breaking bugs. After some extensive sifting and testing with mods, we've pinpointed the problem and it's about time we finally ditched the mod causing the issue.

As stated previously, the mod has become very unstable with it's recent updates, and has been conflicting with many others, so it's highly unlikely that it is the fault of Applied Energistics 2. Thank you so much for your time, and I deeply apologize for not realizing this issue.
